post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc

```diff
@@ -0,0 +1,56 @@
+=  Multiple regions and zones configuration for a cluster on vSphere
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+As an administrator, you can specify multiple regions and zones for your {product-title} cluster that runs on a VMware vSphere instance. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
+
+A failure domain configuration lists parameters that create a topology. The following list states some of these parameters:
+
+* `computeCluster`
+
+* `datacenter`
+
+* `datastore`
+
+* `networks`
+
+* `resourcePool`
+
+After you define multiple regions and zones for your {product-title} cluster, you can create or migrate nodes to another failure domain.
+
+[IMPORTANT]
+====
+If you want to migrate pre-existing {product-title} cluster compute nodes to a failure domain, you must define a new compute machine set for the compute node. This new machine set can scale up a compute node according to the topology of the failure domain, and scale down the pre-existing compute node. 
+
+The cloud provider adds `topology.kubernetes.io/zone` and `topology.kubernetes.io/region` labels to any compute node provisioned by a machine set resource. 
+
+For more information, see xref:../machine_management/creating_machinesets/creating-machineset-vsphere.adoc[Creating a compute machine set].
+====
+
+include::modules/specifying-regions-zones-infrastructure-vsphere.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc#references-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration[Parameters for the cluster-wide infrastructure CRD]
+
+include::modules/vsphere-enabling-multiple-layer2-networks.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../installing/installing_vsphere/installing-vsphere-network-customizations.adoc#installation-network-connectivity-user-infra_installing-vsphere-network-customizations[Network connectivity requirements]
+
+* xref:../machine_management/creating-infrastructure-machinesets.adoc#creating-infrastructure-machinesets-production[Creating infrastructure machine sets for production environments]
+
+* xref:../machine_management/creating_machinesets/creating-machineset-vsphere.adoc#machineset-creating_creating-machineset-vsphere[Creating a compute machine set]
+
+include::modules/references-regions-zones-infrastructure-vsphere.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For an example of a `infrastructures.config.openshift.io` configuration, see xref:../post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc#specifying-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration[Specifying multiple regions and zones for your cluster on vSphere].
```

Comment:
This should also be just the text for the link. Otherwise I think this is good to merge.
```suggestion
* xref:../post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc#specifying-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration[Specifying multiple regions and zones for your cluster on vSphere]
```